core/src/main/java/org/apache/iceberg/rest/ResourcePaths.java

```diff
@@ -74,4 +74,8 @@ public String table(TableIdentifier ident) {
   public String rename() {
     return SLASH.join("v1", prefix, "tables", "rename");
   }
+
+  public String scanReport() {
+    return SLASH.join("v1", prefix, "metrics", "scan");
```

Comment:
Why not make this a table-specific path, like `v1/namespaces/ns/tables/t1/metrics`?

I think it would make more sense to have a table-specific path. Similar to how we're adding `prefix` to the path already, I think it is better for the client to keep track of the table and post metrics to the correct table, instead of using a generic endpoint and making the service decide which table the metrics are for.

Reply:
I did consider both approaches and was 50/50 on which to take and went with the generic endpoint. Making this a table-specific path sgtm. 
However, I would probably consider using `v1/namespaces/{namespace}/tables/{table}/metrics/scan` as the path, since just `metrics` sounds quite generic and doing a `GET` against `metrics` (if we would ever support that) might return completely different metrics than scan metrics. Also we would be future-proof in case we'd want to add some other non-scan related metrics under `v1/namespaces/{namespace}/tables/{table}/metrics/xyz`. Thoughts?